### PR TITLE
[Proposal] Add peek status bar

### PR DESCRIPTION
**Pull request by ChrisBr**
to profile SQL and memcache.
https://github.com/peek/peek.

peek

### CODE REVIEW

File: src/api/app/views/layouts/webui/webui.html.erb [hunk 1]
```diff
@@ -39,6 +39,7 @@
     <%= csrf_meta_tag %>
   </head>
   <body>
+    <%= render 'peek/bar' %>
```
Comment (hennevogel) [inline src/api/app/views/layouts/webui/webui.html.erb]: Hm, always?

Comment (ChrisBr) [inline src/api/app/views/layouts/webui/webui.html.erb]: Default is only in dev

Comment (ChrisBr) [inline src/api/app/views/layouts/webui/webui.html.erb]: But I would also like to render it in production in the future but therefore we need to somehow label users as "staff".

Comment (hennevogel) [inline src/api/app/views/layouts/webui/webui.html.erb]: Which is easy right? Just add a group in seeds...

**Comment by hennevogel** [comment]
@ChrisBr have you packaged the gems?

File: src/api/app/views/layouts/webui/webui.html.erb (hunk 1 above)
Comment (ChrisBr) [inline src/api/app/views/layouts/webui/webui.html.erb]: Yeah, just wasn't sure if this is the right way to go... Will do that then!

**Comment by ChrisBr** [comment]
@hennevogel no haven't packaged yet, will do!

**Comment by ChrisBr** [comment]
@hennevogel updated and packaged the gems https://build.opensuse.org/project/requests/devel:languages:ruby:extensions

File: src/api/app/assets/javascripts/webui/application.js.erb [hunk 2]
```diff
@@ -25,6 +25,8 @@
 //= require cocoon
 //= require moment
 //= require mousetrap
+//= require jquery_ujs
```
Comment (hennevogel) [inline src/api/app/assets/javascripts/webui/application.js.erb]: Why?

File: src/api/db/seeds.rb [hunk 3]
```diff
@@ -55,6 +55,9 @@
                                               realname: "Anonymous User", state: "3",
                                               password: "123456")
 
+puts "Seeding groups table"
```
Comment (hennevogel) [inline src/api/db/seeds.rb]: Erm did I say `Group`? I meant `Role`. Sorry :-/

**Comment by ChrisBr** [comment]
@hennevogel updated

**Pull request merged**